src/collar/oc_anim.lsl

```diff
@@ -511,15 +511,15 @@ UserCommand(integer iNum, string sStr, key kID) {
       string sValueNotLower = llList2String(lParams, 1);
       if (sValue == "on") {
         if (llGetAnimationOverride("Standing") != "") {
-          g_iTweakPoseAO = 1;
+          g_bTweakPoseAO = TRUE;
           llMessageLinked(LINK_SAVE, LM_SETTING_SAVE, g_sSettingToken + "TweakPoseAO=1" , "");
           RefreshAnim();
           llMessageLinked(LINK_DIALOG, NOTIFY, "1" + "AntiSlide is now enabled.", kID);
         } else {
           llMessageLinked(LINK_DIALOG, NOTIFY, "1" + "\n\nAntiSlide can't be used when a server-side AO is already running. If you are wearing the OpenCollar AO, it will take care of this functionality on its own and AntiSlide is not required.\n", kID);
         }
       } else if (sValue == "off") {
-        g_iTweakPoseAO = 0;
+        g_bTweakPoseAO = FALSE;
```

Comment:
I also see how you changed the boolean variables. Nice and thank you for the contribution!